### PR TITLE
Added DrawChar to Bitmap

### PR DESCRIPTION
**Pull request by mikeoliphant**
## Description
This change adds a DrawChar method to the Bitmap class.

## Motivation and Context
See corresponding nf-interpreter PR here: https://github.com/nanoframework/nf-interpreter/pull/2390

## How Has This Been Tested?
Tested locally on M5Stack Core2.

## Types of changes

- [x] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [ ] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [x] My code follows the code style of this project (only if there are changes in source code).
- [ ] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [ ] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [x] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/master/CONTRIBUTING.md) document.
- [x] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

File: nanoFramework.Graphics/Primitive/Bitmap.cs [hunk 1]
```diff
@@ -267,6 +267,17 @@ public void DrawTextInRect(string text, int x, int y, int width, int height, uin
             DrawTextInRect(ref text, ref xRelStart, ref yRelStart, x, y, width, height, dtFlags, color, font);
         }
 
+        /// <summary>
+        /// Draws a single character to the screen
```
Comment (josesimoes) [inline nanoFramework.Graphics/Primitive/Bitmap.cs:271]: Missing period and sentence end.

File: nanoFramework.Graphics/Primitive/Bitmap.cs [hunk 2]
```diff
@@ -267,6 +267,17 @@ public void DrawTextInRect(string text, int x, int y, int width, int height, uin
...
+        /// </summary>
+        /// <param name="c"> The character to draw.</param>
+        /// <param name="x"> The x-coordinate of the upper-left corner to draw to.</param>
+        /// <param name="y"> The y-coordinate of the upper-left corner to draw to.</param>
+        /// <param name="color"> The color to be used for the character.</param>
+        /// <param name="font"> The font to be used for the character.</param>
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        extern public void DrawChar(ushort c, int x, int y, Color color, Font font);
```
Comment (josesimoes) [inline nanoFramework.Graphics/Primitive/Bitmap.cs:279]: Wouldn't the first parameter look better (obvious) being a char? 🤔

Comment (mikeoliphant) [reply nanoFramework.Graphics/Primitive/Bitmap.cs:279]: It is 16bit in the underlying native code (I think to support unicode?) so I kept it as a short in the managed interface.

Comment (josesimoes) [reply nanoFramework.Graphics/Primitive/Bitmap.cs:279]: OK. That makes sense let's leave it this way then.

File: nanoFramework.Graphics/Primitive/Bitmap.cs (hunk 1 above)
Comment (mikeoliphant) [reply nanoFramework.Graphics/Primitive/Bitmap.cs]: Fixed.

Comment (josesimoes) [reply nanoFramework.Graphics/Primitive/Bitmap.cs]: Thanks! Just missing the native version bump now.

**Review by josesimoes — APPROVED** [review]
All good now! Thanks again. 👍🏻

**Pull request merged**